packages/core/examples/editableTextExample.tsx

```diff
@@ -99,11 +87,18 @@ export class EditableTextExample extends BaseExample<IEditableTextExampleState>
                     key="swap"
                     onChange={this.toggleSwap}
                 />,
+                <small key="note">Title field is uncontrolled; report is controlled.</small>,
```

Comment:
I believe we actually moved that note within the editable text placeholder for clarity. I know it might not be the prettiest, but it'll get better soon once we redesign the example frames